manifests/WinDirStat/WinDirStat/1.1.2.yaml

```
@@ -9,8 +9,10 @@ MinOSVersion: 5.1
 License: GNU General Public License, version 2
 LicenseUrl: https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html
 InstallerType: exe
+Silent: /S
+SilentWithProgress: /S
 Installers:
   - Arch: x86
-  - URL: https://www.fosshub.com/WinDirStat.html?dwl=windirstat1_1_2_setup.exe
-  - Sha256: 370a27a30ee57247faddeb1f99a83933247e07c8760a07ed82e451e1cb5e5cdd
+    URL: https://www.fosshub.com/WinDirStat.html?dwl=windirstat1_1_2_setup.exe
+    Sha256: 370a27a30ee57247faddeb1f99a83933247e07c8760a07ed82e451e1cb5e5cdd
```

```suggestion
    Sha256: 370a27a30ee57247faddeb1f99a83933247e07c8760a07ed82e451e1cb5e5cdd
    Switches:
      Silent: /S
      SilentWithProgress: /S
```
Move the switches inside the installer field may help you to pass the validation test.

---

Thank you!  I confess I'm new to this process, this is my first time contributing to a repo like this.  Appreciate the assist!